tools/ARIAtools/computeMisclosure.py

```diff
@@ -159,7 +164,7 @@ def __init__(self,imgfile,workdir='./',
         self.excludePairs = excludePairs
 
         # Other
-        self.verbose = verbose
+        log.setLevel('DEBUG') if self.verbose else ''
```

Comment:
Codacy found an issue: [Expression "log.setLevel('DEBUG') if self.verbose else ''" is assigned to nothing](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6018914)